app/models/purchase.rb

```diff
@@ -97,6 +97,7 @@ def csv_export_attributes
       line_items.total,
       line_items.size,
       amount_spent_in_cents,
+      issued_at.time.strftime("%Y-%m-%d")
```

Comment:
I've never actually seen `.time` before so this is something new to me ;). I think you can achieve the same result without the `.time` and just directly do `issued_at.strftime("%Y-%m-%d")`. Why did you decide to use `.time` in between the issued_at and strftime?

Reply:
To be honest I thought I need to include .time to use strftime. Thank you for pointing that out and I will fix it!